jetty-http/src/test/java/org/eclipse/jetty/http/HttpFieldsTest.java

```diff
@@ -710,15 +749,15 @@ public void testPreventNullField()
         assertThat(fields.size(), is(1));
         ListIterator<HttpField> iter = fields.listIterator();
         iter.next();
-        iter.set(null);
+        iter.set(null); // set field to null - null entry in list now
```

Comment:
@joakime This comment is wrong.  Setting null used to put in a null entry, but now is a noop.